fuel-rxjava/src/main/kotlin/com/github/kittinunf/fuel/rx/RxFuel.kt

```diff
@@ -24,15 +24,15 @@ private fun <T : Any> Request.rx_response(deserializable: Deserializable<T>): Ob
                 when (result) {
```

Comment:
I think in RxJava2, there is `Observable.create` for us to use it safely. Would you mind the whole implementation to use `.create` instead?